engine/src/flutter/shell/platform/darwin/ios/framework/Source/accessibility_bridge_test.mm

```diff
@@ -460,7 +460,9 @@ - (void)testReplacedSemanticsDoesNotCleanupChildren {
     flutter::SemanticsNode new_node;
     new_node.id = 1;
     new_node.rect = SkRect::MakeXYWH(0, 0, 100, 200);
-    new_node.flags = static_cast<int32_t>(flutter::SemanticsFlags::kHasImplicitScrolling);
+    FlutterSemanticsFlags flags;
+    new_node.flags = flutter::SemanticsFlags2{};
```

Comment:
Can we make it so that a new node is initialized with an empty set of flags as part of construction? This would remove the `new_node.flags = flutter::SemanticsFlags2{}` boilerplate that appears a lot here and below.